### PR TITLE
docs: updates developer documentation

### PR DESCRIPTION
**Pull request by awentzel**
<body>
Migrates content from fluentweb.com that is particular to FAST.

<footer>
 

For [details on formatting](https://fluentweb.com/prototyping/contribution-policy#pull-request-guidance) pull requests.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -3,4 +3,189 @@ A “Contribution” is any work that is voluntarily submitted to a project. Thi
...
+
+It is vital that users trust the contributors to have their best interests in mind while making decisions affecting the overall direction. This means having defined guidelines, a clear roadmap, and a predictable release schedule.
+
+Adoption of new capabilities and features must be carefully balanced by the expressed needs of the users and the community. Change for the sake of change must be avoided.
+
+In order to best serve the community as a whole, the development, release, and work management processes must reflect these guiding principles.
+
+## Governance
+### Steering Committee
+The Fluent Web Steering Committee ("Steering Committee"), membership consists of key Fluent Web Collaborators ("Collaborators") who have demonstrated both design and technical expertise critical to the ongoing support, maintenance, evolution, and a long term commitment to driving the project and community forward.
```
Comment (chrisdholt) [inline CONTRIBUTING.md]: This paragraph (and elsewhere) should probably not reference FW in the mono-repo.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -3,4 +3,189 @@ A “Contribution” is any work that is voluntarily submitted to a project. Thi
...
+
+Collaborators can be nominated to become members of the Steering Committee following the same nomination and approval model. However, Collaborators cannot nominate themselves for Steering Committee membership. A Contributor cannot be nominated and accepted into the Steering Committee without first having been nominated and accepted as a Collaborator.
+
+### Owners
+Fluent Web Owners ("Owners"), having 'admin' access, have ultimate authority and are responsible for management, maintainenance, and operations of repositories.
```
Comment (chrisdholt) [inline CONTRIBUTING.md]: Should we reference FW here? Seems like this would be specific to the -msft package

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -3,4 +3,189 @@ A “Contribution” is any work that is voluntarily submitted to a project. Thi
...
+
+### Collaborators
+Fluent Web Collaborators ("Collaborators"), having 'write' access, have major impact on the project and participate in triaging issues, reviewing code, mentoring, and working to improve and maintain overall architecutural quality.
```
Comment (chrisdholt) [inline CONTRIBUTING.md]: FW?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -3,4 +3,189 @@ A “Contribution” is any work that is voluntarily submitted to a project. Thi
...
+
+Contributors can be nominated as Collaborators by Steering Committee members. Once the nomination is approved by the Steering Committee, the invitation to become a Collaborator is extended to the individual. Assuming the individual accepts the invitation and agrees to the terms of DCO (Developer Certificate of Origin) they are granted commit-access to the project.
+
+Nominations for Collaborator status happen through the typical Steering Committee decision making process. That is, to nominate one or more Collaborators, Steering Committee then moves to interview the nominees. The nominee is approved or rejected following the same consensus seeking process for new team members.
+
+### Contributors
+Fluent Web Contributors ("Contributors"), having 'read' access, can be any individual who have shown interest by contributing a pull request to the project.
```
Comment (chrisdholt) [inline CONTRIBUTING.md]: Same here

File: README.md [hunk 5]
```diff
@@ -1,45 +1,63 @@
...
+
+#### FAST components styled with for Microsoft
+A set of React components with a Microsoft design system driving the look and feel. [Readme](https://github.com/Microsoft/fast-dna/tree/master/packages/fast-components-react-msft)
+
+#### FAST development site
+A development site which can be used to create a React environment for a group of React or web components. [Readme](https://github.com/Microsoft/fast-dna/tree/master/packages/fast-development-site-react)
+
+
+#### FAST JSS manager
+A React higher order component (HOC) library for managing component JSS stylesheets. [Readme](https://github.com/Microsoft/fast-dna/tree/master/packages/fast-jss-manager-react)
```
Comment (chrisdholt) [inline README.md]: should 'higher order' be hyphenated?

File: CONTRIBUTING.md (hunk 1 above)
Comment (awentzel) [reply CONTRIBUTING.md]: I considered changing this, however, it directly aligns with sharing names with our security and user group policy so I want to keep for now.

File: CONTRIBUTING.md (hunk 2 above)
Comment (awentzel) [reply CONTRIBUTING.md]: same as above

File: CONTRIBUTING.md (hunk 3 above)
Comment (awentzel) [reply CONTRIBUTING.md]: ditto

File: CONTRIBUTING.md (hunk 4 above)
Comment (awentzel) [reply CONTRIBUTING.md]: ditto

File: README.md (hunk 5 above)
Comment (awentzel) [reply README.md]: fixed

**Pull request merged**